.github/actions/occt-install-dep/action.yml

```diff
@@ -68,6 +85,33 @@ runs:
         ${{ runner.os == 'macOS' && '-DCMAKE_BUILD_WITH_INSTALL_RPATH=ON -DCMAKE_OSX_DEPLOYMENT_TARGET=12.0' || null }}
         ${{ runner.os == 'Windows' && '-Ax64 -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreadedDLL' || null }}
 
+    - name: Configure OCCT min version
```

Comment:
Should we just drop old OCCT versions now in order to simplify the CI and the F3D plugin code too?